javascripts/content_scripts/privly.js

```diff
@@ -390,6 +390,9 @@ var privly = {
     //Set the source URL
     iFrame.setAttribute("src", applicationUrl);
 
+    //URL which it will be replacing
+    iFrame.setAttribute("data-privlyHref",object.getAttribute("data-privlyHref"));
```

Comment:
@smcgregor In which field is it set? I checked the whole iframe element by going through it, the original link wasn't there. So, I added this and have used it.